src/Elastic.Apm/Api/SpanContext.cs

```diff
@@ -5,14 +5,15 @@
 using System;
 using System.Collections.Generic;
 using Elastic.Apm.Helpers;
+using Elastic.Apm.Model;
 using Elastic.Apm.Report.Serialization;
 using Newtonsoft.Json;
 
 namespace Elastic.Apm.Api
 {
 	public class SpanContext
```

Comment:
Should `SpanContext` expose `SetLabel()` methods in order to set labels of different types, that `Span.SetLabel()` methods should call? In other words, are there usages where labels might be set on `SpanContext` outside of the scope of a `Span`? There are some in unit tests

Reply:
Same as https://github.com/elastic/apm-agent-dotnet/pull/982#discussion_r515903177